src/modules/45-dashboards/pages/__tests__/DashboardsCommonHeaderPage.test.tsx

```diff
@@ -71,4 +73,16 @@ describe('DashboardsHeader', () => {
 
     await waitFor(() => expect(screen.queryByText(sidebarQueryText)).toBeInTheDocument())
   })
+
+  test('it should close the getting started sidebar when clicking the getStarted X button', async () => {
+    renderComponent()
+    const getStartedButton = screen.getByText('getStarted') as HTMLButtonElement
+    userEvent.click(getStartedButton)
+    await waitFor(() => expect(screen.getByText('cross')).toBeInTheDocument())
+
+    const closeButton = screen.getByText('cross') as HTMLButtonElement
+    userEvent.click(closeButton)
```

Comment:
Would `getByRole` work here?
```suggestion
    userEvent.click(screen.getByRole('button', { name: 'cross' }))
```